src/main.ts

```diff
@@ -18,6 +18,9 @@ const useSwagger = (app: INestApplication) => {
 
 const bootstrap = async () => {
   const app = await NestFactory.create(AppModule);
+  app.enableCors({
```

Comment:
please add condition to enable only if : !process.env.NODE_ENV || process.env.NODE_ENV !== 'production'